operations/helm/charts/mimir-distributed/RELEASE.md

```diff
@@ -2,11 +2,20 @@
 
 The release processes that follow apply to the Grafana Mimir Helm chart.
 
+## Schedule
+
+The release schedule follows the Grafana Mimir and Grafana Enterprise Metrics (GEM) releases so that a new Helm chart release will include both. This usually happens within 2 weeks of the Mimir release. For the Mimir release schedule consult the Mimir [RELEASE.md](https://github.com/grafana/mimir/blob/main/RELEASE.md).
```

Comment:
```suggestion
The release schedule follows the Grafana Mimir and Grafana Enterprise Metrics (GEM) releases so that a new Helm chart release will include both. This usually happens within 2 weeks of the Mimir release. For the Mimir release schedule consult the Mimir [RELEASE.md](../../../../RELEASE.md).
```

nitpick so that when viewing this locally you don't open a browser